MAUI/DataGrid/Column-types.md

```diff
@@ -127,8 +127,117 @@ public class DisplayBindingConverter : IValueConverter
 
 In order to set the TextAlignment of the header cell and data row cell , use the [DataGridColumn.HeaderTextAlignment](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_HeaderTextAlignment) and [DataGridColumn.CellTextAlignment](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellTextAlignment) property. The default text alignment is based on the type of the columns. The header and data rows are right aligned for numeric, date columns and left aligned for text column.
 
-### Header customizations
+### Line Break Mode
```

Comment:
**Add the line break mode topic after Formatting sub topic.**

The following content only will be enough for line break mode topic.

### Line break mode

The text wrapping and truncation of a column's cell values can be customized by setting the `DataGridColumn.LineBreakMode` property.
 
  {% tabs %}
{% highlight xaml tabtitle="MainPage.xaml" %}
<syncfusion:SfDataGrid AutoGenerateColumnsMode="None"
                       ItemsSource="{Binding OrderInfo}">
    <syncfusion:SfDataGrid.Columns >
        <syncfusion:DataGridTextColumn MappingName="Name" LineBreakMode="NoWrap" />
    </syncfusion:SfDataGrid.Columns>
</syncfusion:SfDataGrid>
{% endhighlight %}
{% endtabs %}

N> The truncation modes will not work on the Windows platform.